codecov.yaml

```diff
@@ -1,53 +1,75 @@
+#  # `Header` is the top of the comment showing the two commits being merged, the change in coverage, and the diff coverage.
+#  # `Footer` is the bottom of a comment that contains the legend, last updated commits, and docs link.
+#  layout: "reach,diff,flags,tree,header,footer"
+#  # The way Codecov submits comments in your pull requests.`default` means update if exists.Otherwise, post new.
+#  behavior: default
+#  # If `require_changes` set true, comments will now only post when coverage changes.
+#  # Furthermore, if a comment already exists, and a newer commit results in no coverage
+#  # change for the entire pull, the comment will be deleted.
+#  require_changes: no
+#  # Force comments to post on pull requests even if Codecov doesn't
+#  # have coverage reports for either the base or head commit.
+#  require_base: no
+#  require_head: no
+#  # Codecov does not comment until at least 5 builds have been uploaded from the CI pipeline.
+#  # By default, codecov will post and/or update the pull request comment after it processes
+#  # each report uploaded for a particular pull request commit.
+#  # As the Antrea CI process uploads many reports to codecov, this can be confusing for
+#  # team members viewing the pull request as reports are processing.
+#  after_n_builds: 5
+#  # Carryforward Flags are designed for projects that do not upload total coverage for every commit.
+#  show_carryforward_flags: false
 
 github_checks:
+  # Specify whether to use GitHub Checks annotations or normal statuses.
   annotations: true
 
 coverage:
   status:
     patch:
       default:
+        # Choose a minimum coverage ratio that the commit must meet to be considered a success.
+        # <number> specify a target of an exact coverage number.
         target: 70%
+        # Allow the coverage to drop by X%, and posting a success status.
         threshold: 5%
+        # Use the pull request base if the commit is on a pull request. If not, the parent commit will be used.
         base: auto
+        # Only post a status to pull requests.
         only_pulls: true
-        flags:
-          - unit-tests
     project:
       default:
+        # `auto` will use the coverage from the base commit (pull request base or parent commit) coverage to compare against.
         target: auto
       antrea-unit-tests:
         target: auto
+        # Flags are a list of user defined Flags, and the impact on their coverage.
         flags:
           - unit-tests
-      antrea-integration-tests:
-        target: auto
-        flags:
-          - integration-tests
-      antrea-e2e-tests:
-        target: auto
-        flags:
-          - e2e-tests
-      antrea-kind-e2e-tests:
-        target: auto
-        flags:
-          - kind-e2e-tests
+
 
 flag_management:
   default_rules:
-    carryforward: true
+    carryforward: false
+ # individual_flags:
+ #   - name: unit-tests
+ #     carryforward: false
```

Comment:
Hi @wenqiq  I didn't recall that we plan to disable `carryforward`? any clue for these changes?

Reply:
@wenqiq any comment for this?

Reply:
After verifying this and I think `carryforward` should enable here. I will update this.